docs/resources/v2-conformance-statement.md

```diff
@@ -173,6 +174,70 @@ An example response with `Accept` header `application/dicom+json`:
 }
 ```
 
+An example response with `Accept` header `application/dicom+json` with a FailedAttributesSequence in a ReferencedSOPSequence:
+
+```json
+{
+  "00081190":
+  {
+    "vr":"UR",
+    "Value":["http://localhost/studies/d09e8215-e1e1-4c7a-8496-b4f6641ed232"]
+  },
+  "00081199":
+  {
+    "vr":"SQ",
+    "Value":
+    [{
+      "00081150":
+      {
+        "vr":"UI",
+        "Value":["d246deb5-18c8-4336-a591-aeb6f8596664"]
+      },
+      "00081155":
+      {
+        "vr":"UI",
+        "Value":["4a858cbb-a71f-4c01-b9b5-85f88b031365"]
+      },
+      "00081190":
+      {
+        "vr":"UR",
+        "Value":["http://localhost/studies/d09e8215-e1e1-4c7a-8496-b4f6641ed232/series/8c4915f5-cc54-4e50-aa1f-9b06f6e58485/instances/4a858cbb-a71f-4c01-b9b5-85f88b031365"]
+      },
+      "00081196": {
+        "vr": "US",
+        "Value": [
+            1
+        ]
+      },
+      "00741048": {
+        "vr": "SQ",
+        "Value": [
+          {
+            "00000902": {
+              "vr": "LO",
+              "Value": [
+                "DICOM100: (0008,0020) - Content \"NotAValidDate\" does not validate VR DA: one of the date values does not match the pattern YYYYMMDD"
+              ]
+            }
+          },
+          {
+            "00000902": {
+              "vr": "LO",
+              "Value": [
+                "DICOM100: (0008,002a) - Content \"NotAValidDate\" does not validate VR DT: value does not mach pattern YYYY[MM[DD[HH[MM[SS[.F{1-6}]]]]]]"
+              ]
+            }
+          }
+        ]
+      }
+    }]
+  }
+}
+```
+
+In the case that leniency is applied and the file is stored, you will be notified with each failing attribute per instance. In case of a failing sequence or multiple issues of a single attribute, only the first failing attribute reason will be noted.
+We will also output diagnostic logs per failed attribute per instance.
```

Comment:
I think we should avoid using "we" here.  I know "we" is used in other places, but it would be a good practice start changing.  

```suggestion
Diagnostic logs will also list failed attributes per instance.
```